src/components/promoCode/services/promoCodeSrv.service.js

```diff
@@ -2,8 +2,15 @@
     'use strict';
 
     angular.module('znk.infra-web-app.promoCode').provider('PromoCodeSrv',
-        function () {
+        function (ENV) {
+
             var backendData = {};
+            var appContext = ENV.firebaseAppScopeName;
+
+            backendData[appContext] = {  //default data
```

Comment:
there are 2 places using promo code srv, and the use is different:

1.in website - i need the all apps env and there i initializing the backendData[appContext].firebaseAppScopeName (in config phase). thats the reason i need the appContext map.
2.in app (student/dashboard) - i use the default backendData and i need only the appName (it is the student fb path).